db/seeds.dev.rb

```diff
@@ -104,6 +104,7 @@ def create_policy(account, canonical_profile, hosts)
         host.os_minor_version
       )
 
+      profile.rules = canonical_profile&.rules&.first(2) if profile&.rules&.empty?
```

Comment:
I think we can fine-tune this a bit more, but for now this is good if @xmicha82 agrees.

Reply:
Yeah, let me know if you have any suggestions. I was trying to think of the simplest way to assign a couple of rules to profiles that didn't have any rules

Reply:
LGTM!